head/src/main/kotlin/io/qalipsis/core/head/campaign/CampaignManager.kt

```diff
@@ -46,4 +47,12 @@ internal interface CampaignManager {
      * @param hard force the campaign to fail when set to true, defaults to false
      */
     suspend fun abort(aborter: String, tenant: String, campaignKey: String, hard: Boolean)
+
+    /**
+     * Replay a campaign with the provided name.
+     * @param configurer is a username of the user replaying the campaign
+     * @param tenant define in which tenant the abortion of the campaign should be done
+     * @param campaignKey is a name of the campaign to abort
+     */
+    suspend fun replay(tenant: String, campaignKey: String, configurer: String): Campaign
```

Comment:
```suggestion
    suspend fun replay(tenant: String, configurer: String, campaignKey: String): RunningCampaign
```
Please keep the signatures of the different signatures consistent (which makes me realize that the abort is wrong as well). Could you please reorder the arguments properly?